src/i18n/en-US/index.ts

```diff
@@ -169,6 +169,8 @@ export default {
     unbonding: 'Unbonding',
     myDapps: 'My dApps',
     dappRegistered: 'Congrats!! your contract is approved. Please submit the details',
+    welcomeBanner:
+      'Congraturations 🎉 Please fill in the dApp information as much as possible to give users a good information. Your card will be apear in dApps Staking Portal shortly after you submit the form.',
```

Comment:
How about change it like this? @bobo-k2  @Kahonnohak @hoonsubin 

'Congratulations 🎉 Please fill in your dApp information to give users better information. Updated information will appear on the dApp page shortly.'

Reply:
@impelcrypto I agree that current welcome sentence could be improved. I would like to slightly update your sentence since word information is used too many times. 

'Congratulations 🎉 Please fill in your dApp information to give users better overview of your application. Updated data will appear on the dApp page shortly.'

cc @Kahonnohak @hoonsubin